app/src/main/kotlin/com/camerasecuritysystem/client/models/DoesNetworkHaveInternet.kt

```diff
@@ -0,0 +1,25 @@
+package com.camerasecuritysystem.client.models
+
+import android.util.Log
+import java.io.IOException
+import java.net.InetSocketAddress
+import javax.net.SocketFactory
+
+
+object DoesNetworkHaveInternet {
+    // Make sure to execute this on a background thread.
+    fun execute(socketFactory: SocketFactory): Boolean {
+        return try{
+            Log.d("PING", "PINGING google.")
+            val socket = socketFactory.createSocket() ?: throw IOException("Socket is null.")
+            socket.connect(InetSocketAddress("8.8.8.8", 53), 1500)
```

Comment:
We still want to ping google? Is it possible to ping the IP address of the server?

Reply:
Noted. I made a ToDo card in Trello

Reply:
Is on the roadmap